src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let filterMan = people.filter(item => item['sex'] === 'm');
+
+  if (century) {
+    filterMan = filterMan
+      .filter(item => Math.ceil(item['died'] / 100) === century);
+  }
+
+  return filterMan.reduce((sum, { born, died }) => sum + died - born, 0)
+    / filterMan.length;
```

Comment:
Create a function for the calculation of average age. You do the same actions everywhere